api/src/db/model/request.ts

```diff
@@ -72,7 +104,8 @@ export default class RequestModel extends Model {
         const query = {
             text: `UPDATE ${this.table}
             SET
-            profile_id = $1, edit_type = $2, edit_object = $3, nats_subject = $4, nats_context = $5
+            profile_id = $1, edit_type = $2, edit_object = $3, type = $4,
+            requires_human_action = $5,is_active = $6, user_id = $7
```

Comment:
I **think** you can put `WHERE id = $8` here and then just add an 8th item to the `values` array here. Generally considered more secure in SQL.